org.eclipse.jdt.debug.ui/plugin.properties

```diff
@@ -337,3 +337,6 @@ OpenFromClipboardAction.description = Opens a Java element or a Java stack trace
 OpenFromClipboardAction.name = Open from Clipboard
 
 VariablesView.name = Variables
+
+CompareObjects.label=Compare Objects
```

Comment:
I think mentioning "Objects" is bad. Doesn't it work for primitives? How about Compare "Values" or just "Compare" only? Will it work for results of Expressions in the expressions view too?

Reply:
It wont work with primitives, only with variable data that are accessible from variables view. 
,will change it "Compare" like you suggested.
I haven't added this in expressions view